images/win/scripts/Installers/Install-PyPy.ps1

```diff
@@ -74,7 +74,6 @@ function Install-PyPy
         # PyPy v7.3.1 or higher creates only Scripts folder therefore to preserve back compatibility with UsePythonVersionV0 task
         # We should create a Scripts -> bin symlink
         Write-Host "Symbolic link created for '$pypyArchPath\Scripts' <<===>> '$pypyArchPath\bin'"
-        New-Item -Path "$pypyArchPath\bin" -ItemType SymbolicLink -Value "$pypyArchPath\Scripts" | Out-Null

```

Comment:
I think we should also remove Write-Host on 76 line and comment above since it is not actual anymore